pyaedt/desktop.py

```diff
@@ -1816,43 +1825,47 @@ def submit_job(
                 return False
         else:
             if not os.path.exists(aedt_full_exe_path):
-                self.logger.error("AEDT shared path does not exist. Provide a full path.")
-                return False
+                self.logger.warning("The AEDT executable path not visible from the client.")
```

Comment:
Is this client or server? Shouldn't the exe be visible on the server?

Reply:
yes that is the server, so the client not necessarly is able to see it. That's why we cannot force the return False if we are not able to see it from the client